docs/gallery/python/2_average_expression.py

```diff
@@ -0,0 +1,299 @@
+# across various organs and cell types.
+#
+# Adapt this code to explore gene expression with the `atlasapprox <https://atlasapprox.readthedocs.io/en/latest/index.html>`_
+# API using your genes and organs of interest.
+
+# %%
+# .. _similar-features:
+#
+# Exploring genes with similar expression patterns
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Building on the analysis of average gene expression across organs, this
+# section introduces the ``similar_features`` method to explore related gene
+# expression, using *ACTB* as an example gene of interest.
+#
+# This method helps answer questions like "Which genes exhibit similar
+# expression patterns to a chosen gene in this organ?" and retrieves the top 10
+# similar genes, as shown below:
+
+similar_features = api.similar_features(
+    organism='h_sapiens',
+    organ='lung',
+    feature='ACTB',
+    method='correlation',
+    number=10
+)
+
+# Display the result
+similar_features
+
+# %%
+# The ``similar_features`` method returns a **pandas.Series** with gene names as
+# the index and their Pearson correlation coefficients relative to the gene of
+# interest (e.g., *ACTB*) as values.
+# These coefficients, which range from -1 (perfect negative correlation) to 1
+# (perfect positive correlation), indicate how similar each gene's expression
+# pattern is to that of the target gene.
+#
+# For example, *LASP1* has the highest similarity to *ACTB* (0.483309),
+# suggesting potential co-regulation across cell types in the lung.
+
+# %%
+# Additionally, other than correlation, methods like euclidean and
+# manhattan are also available. Please refer to the `atlasapprox <https://atlasapprox.readthedocs.io/en/latest/index.html>`_
+# API documentation for details.
+
+# %%
+# Visualise the expression profile of these genes
+# -----------------------------------------------
+# To visualize the expression profiles of these genes, use a heatmap by
+# extracting gene names from ``similar_features.index`` and including the gene
+# of interest (e.g., *ACTB*) for comparison.
+#
+# Pass these as the ``features`` parameter to the ``average`` method, then plot
+# the results:
+
+# Combine ACTB and its similar features into a single list for better comparison
+gene_list = ['ACTB'] + list(similar_features.index)
+
+# Get average gene expression
+avg_similar_features = api.average(
+    organism='h_sapiens',
+    organ='lung',
+    features=gene_list
+)
+
+# Display the heatmap
+fig, ax = plt.subplots(figsize=(8, 6))
+heatmap = sns.heatmap(avg_similar_features,ax=ax)
+
+# Customize labels
+plt.title(f'Average gene expression across ACTB and its similar features')
+plt.xlabel("Cell type")
+plt.ylabel("Gene")
+
+fig.tight_layout()
+
+# %%
+# .. _marker-genes:
+#
+# Finding cell-type-specific markers in an organ
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# So far, the focus has been on known genes or those similar to a gene of interest.
+# But what if no specific gene comes to mind?
```

Comment:
This is sloppy, colloquial English and inappropriate. Please rephrase the intro?